src/models/image-map.test.ts

```diff
@@ -94,11 +94,15 @@ describe("ImageMap", () => {
 
   it("test localAssetsImagesHandler", () => {
     expectToMatch(localAssetsImagesHandler, [kLocalImageUrl]);
-    return localAssetsImagesHandler.store(kLocalImageUrl)
+    localAssetsImagesHandler.store(kLocalImageUrl)
             .then(storeResult => {
               expect(storeResult.contentUrl).toBe(kLocalImageUrl);
-              expect(storeResult.displayUrl).toBe(`${kCurriculumBaseUrl}/${kLocalImageUrl}`);
+              expect(storeResult.displayUrl).toBe(`${kCurriculumBaseUrl}branch/main/${kLocalImageUrl}`);
             });
+    localAssetsImagesHandler.store("curriculum/stretching-and-shrinking/images/image.png")
+             .then(storeResult => {
+               expect(storeResult.displayUrl).toBe(`${kCurriculumBaseUrl}branch/main/sas/images/image.png`);
+             });
```

Comment:
I think the promise needs to be returned here so Jest knows to wait for it before finishing the test.
I'm not sure the best way to do that in this case. You could chain another `then` off of the first promise, but that would also have to return its promise.

It is probably better to switch to an async function. Then you can use await for the `store` calls. 

More information is here: https://jestjs.io/docs/asynchronous